romana/cmd/tenant.go

```diff
@@ -91,7 +93,7 @@ var tenantDeleteCmd = &cli.Command{
 
 func tenantCreate(cmd *cli.Command, args []string) error {
```

Comment:
This function really should have a docstring: What does it do, what is it for, who uses it, etc.